### PR TITLE
Wrap the error in user and product update

### PR DESCRIPTION
**Pull request by haibin**
https://github.com/ardanlabs/service/issues/141

### CODE REVIEW

File: business/data/user/user.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func (u User) Update(ctx context.Context, traceID string, claims auth.Claims, us
 
 	usr, err := u.QueryByID(ctx, traceID, claims, userID)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "updating user")
```
Comment (ardan-bkennedy) [inline business/data/user/user.go:93]: We need to do this as well with product for the update.

Comment (haibin) [reply business/data/user/user.go:93]: done

**Comment by ardan-bkennedy** [comment]
Do one more thing. Add your name and email to the contributors file. Keep things in alphabetical order please.

**Comment by ardan-bkennedy** [comment]
Funny, we should have ran the tests first. I had to make a change to the handlers to use errors.Cause. Missing this was a mistake as well.

**Comment by haibin** [comment]
> Funny, we should have ran the tests first. I had to make a change to the handlers to use errors.Cause. Missing this was a mistake as well.

I see. We probably should integrate tests in the CI/CD.

**Comment by ardan-bkennedy** [comment]
I do, that is how I knew there was a problem with the change. :)

**Comment by ardan-bkennedy** [comment]
Maybe they are not being run outside of master?

**Comment by haibin** [comment]
I don't see it run for PRs. We can do it for PRs.

**Comment by ardan-bkennedy** [comment]
Ok, I will look for that setting!!

**Comment by ardan-bkennedy** [comment]
PR's are checked for the webhook. Uhm?

**Comment by haibin** [comment]
You can take a look at this https://circleci.com/docs/2.0/oss/#build-pull-requests-from-forked-repositories

**Comment by ardan-bkennedy** [comment]
Ok Done!

**Pull request merged**